### ISSUE TITLE
Noise filter makes Graphics instances black (lose color)

### ISSUE THREAD

**Issue by UltraCakeBakery** [pixijs/pixi-filters#243]
hi,

I added the noise filter from "@pixi/filter-noise" to the main stage and all my graphics are now black:

![image](https://user-images.githubusercontent.com/12383587/74420573-f51c2e80-4e4b-11ea-9093-55e5edfeba40.png)

```
import { Container } from 'pixi.js-legacy'
import { NoiseFilter } from '@pixi/filter-noise'

export default class MainStage extends Container
{
    private ticker: PIXI.Ticker

    constructor( ticker: PIXI.Ticker )
    {
        super()

        this.ticker = ticker
        this.sortableChildren = true

        // Noise Filter
        const noiseFilter = new NoiseFilter({ noise: 0.04 })
        ticker.add(() => { noiseFilter.seed = Math.random() })

        this.filters = [noiseFilter];
    }
```

This is running in the latest version of chrome using WebGL 2.0. Sadly need to support canvas so thats why I use the legacy build.

**Comment by bigtimebuddy** [comment]
@UltraCakeBakery could you please provide your resolution? It’s helpful for people who bump into this later.

**Comment by UltraCakeBakery** [comment]
Sorry, I must have accidentally removed the message out of joy 😅.
pixijs-legacy has its own filter package (pixi.js-legacy/filters). I imported that one instead of pixi-filters. It has a limited selection of filters but works as expected.

I now switched to the non-legacy version of PIXI and the filters from pixi-filters work perfectly normal now. So yeah, I basically changed the imported package (currently not at home and responding from my phone, code may not be accurate);

```
import { Container } from '@pixi'
import { NoiseFilter } from 'pixi.js-legacy/filters'
```

I will switch back to the legacy version later on. I definitely want the fallback support.
